mitodl/open-discussions
# Ab/add rss to subscribe button

#### Pre-Flight checklist

<img width="381" alt="Screen Shot 2020-08-03 at 2 06 37 PM" src="https://user-images.githubusercontent.com/1934992/89213138-9fbb1180-d592-11ea-9488-b512c49d9091.png">
<img width="378" alt="Screen Shot 2020-08-03 at 2 06 30 PM" src="https://user-images.githubusercontent.com/1934992/89213140-a053a800-d592-11ea-8c6e-7554d8470cb8.png">
<img width="1602" alt="Screen Shot 2020-08-03 at 2 06 09 PM" src="https://user-images.githubusercontent.com/1934992/89213141-a053a800-d592-11ea-80be-015e866ca53e.png">
<img width="1583" alt="Screen Shot 2020-08-03 at 2 06 00 PM" src="https://user-images.githubusercontent.com/1934992/89213145-a21d6b80-d592-11ea-8ce1-edda482f3350.png">
<img width="375" alt="Screen Shot 2020-07-30 at 5 48 53 PM" src="https://user-images.githubusercontent.com/1934992/88978220-46e73280-d28d-11ea-9a85-f4561a4bd56f.png">
<img width="484" alt="Screen Shot 2020-07-30 at 5 48 39 PM" src="https://user-images.githubusercontent.com/1934992/88978222-477fc900-d28d-11ea-8b43-f164998b78f1.png">

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/3094
closes https://github.com/mitodl/open-discussions/issues/3085

#### What's this PR do?
Adds rss to a the subscribe button for individual podcasts
Adds a subscribe button for the aggregate podcast feed rss

#### How should this be manually tested?
Verify that the subscribe button for individual podcasts has a rss link and that the rss links work
Verify that there is a subscribe button for the MIT aggregate rss feed and the link works
